paying_for_college/config/settings/base.py

```diff
@@ -43,10 +44,10 @@
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
-    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
+    # 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
-    'django.middleware.security.SecurityMiddleware',
+    # 'django.middleware.security.SecurityMiddleware',
```

Comment:
we have to remove it, because django 1.6 can't use it and bombs.  
i leave it commented because I hope to return to 1.8 if a path to production opens up.  
also, these settings only affect the stand-alone version and are ignored elsewhere.